pixeltable/tool/create_test_db_dump.py

```diff
@@ -61,7 +61,7 @@ def dump_db(self) -> None:
         info_dict = {'pixeltable-dump': {
             'metadata-version': md_version,
             'git-sha': git_sha,
-            'datetime': datetime.datetime.utcnow(),
+            'datetime': datetime.datetime.now(tz=datetime.timezone.utc),
```

Comment:
comment why not utcnow() here

Reply:
It's deprecated as of Python 3.12. Does that really need a comment?